README.md

```diff
@@ -80,6 +80,11 @@ the only label value published for features is the string `"true"`._
 
 The `--sources` flag controls which sources to use for discovery.
 
+_Note: consecutive runs of node-feature-discovery will update the labels on a
+given node. If labels are not discovered on a consecutive run they will be removed
```

Comment:
s/If labels are not discovered on a consecutive run they will be removed/If features are not discovered on a consecutive run, the corresponding label will be removed